src/UseCases/DisplayPage/PageContentModifier.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+namespace WMDE\Fundraising\Frontend\UseCases\DisplayPage;
+
+use Psr\Log\LoggerInterface;
+
+/**
+ * @licence GNU GPL v2+
+ */
+class PageContentModifier {
+
+	private $logger;
+	private $substitutions;
+
+	public function __construct( LoggerInterface $logger, array $substitutions = [] ) {
+		$this->logger = $logger;
+		$this->substitutions = $substitutions;
```

Comment:
Actually this thing is of type `FreshCacheList`, not `array`. I've added a commit on top of this PR commenting the code out.